Initial Example Code

AllenVan opened the pull request:
Raul please

AllenVan committed:
Bare bones basic bot.

AllenVan committed:
guess who accidentally posted my bot token privately

AllenVan committed:
guess who forgot to add a newline at the end

RaulCard commented:
write a better title for the PR

RaulCard reviewed (changes requested):
Add missing steps to the README for setting up `tutorial-env`. Add scripts dir to gitignore so these environment files are not included.

RaulCard commented on example_bot.py:
```diff
@@ -0,0 +1,53 @@
+embed=discord.Embed(title="Example", url="https://i.redd.it/4rb49b4ot6ra1.png", description="Sample Embed", color=0x109319)
+
+# Add author, thumbnail, fields, and footer to the embed
+embed.set_author(name="RaulCard", url="https://github.com/RaulCard", icon_url="https://avatars.githubusercontent.com/u/31727722?v=4")
```
why my name here 💀

AllenVan replied:
If you don't like it make a new pull request fixing it

AllenVan committed:
Removed sensitive info and improved ReadMe

AllenVan commented:
-Removed all environment directories
-Updated ReadMe to have comprehensive instructions to create virtual environment
-Added bot-env/ directory to .gitignore

AllenVan committed:
Added a newline in the ReadMe

AllenVan committed:
Modified ReadMe again for creating env

RaulCard commented on example_bot.py:
```diff
@@ -0,0 +1,56 @@
+
+
+@client.event
+async def on_ready():
```
This is overwritten. Combine the two `on_ready()` functions or delete one.

RaulCard commented on example_bot.py:
```diff
@@ -0,0 +1,56 @@
+    print(f'We have logged in as {client.user}')
+
+@client.event
+async def on_message(message):
```
It aint working for me. Maybe there is an intent we need.

RaulCard replied:
Ah, bro you overwrite this function too
